airbyte-integrations/connectors/source-microsoft-lists/metadata.yaml

```
@@ -26,6 +26,14 @@ data:
   releaseDate: 2024-10-18
   releaseStage: alpha
   supportLevel: community
+  connectorTestSuitesOptions:
+    - suite: integrationTests
```

It seems Manifest-Only connectors don't have an "Integration Test" bucket. Also, since they are living in the `unit_tests` folder, I believe the below is correct:

```suggestion
    - suite: unitTests
```

---

Me dumb, just fixed. Thanks for catching.